runners/flink/job-server/flink_job_server.gradle

```diff
@@ -89,6 +89,7 @@ dependencies {
   // For resolving external transform requests
   runtime project(":sdks:java:io:kafka")
   runtime library.java.kafka_clients
+  runtime project(":sdks:java:io:snowflake")
```

Comment:
I could remove it when I was writing jdbcio for xlang

Reply:
Done , thanks :)